app/views/waste_exemptions_engine/confirmation_mailer/send_confirmation_email.html.erb

```diff
@@ -15,113 +15,113 @@
   <title><%= WasteExemptionsEngine.configuration.service_name %></title>
 </head>
 
-<body style="font-family: Helvetica, Arial, sans-serif; font-size: 16px; margin: 0; color: #0b0c0c">
-  <table width="100%" cellpadding="0" cellspacing="0" border="0">
-    <tr>
-      <td width="100%" height="53px" bgcolor="#0b0c0c">
-        <table width="580" cellpadding="0" cellspacing="0" border="0" align="center">
-          <tr>
-            <td width="100%" bgcolor="#0b0c0c" valign="middle">
-              <%= email_image_tag("govuk_logotype_email.png", { alt: 'GOV.UK', border: '0' }) %>
-            </td>
-          </tr>
-        </table>
-      </td>
-    </tr>
-  </table>
-  <table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#FFFFFF">
-    <tr>
-      <td width="100%">
-        <table width="580" cellpadding="0" cellspacing="0" border="0" align="center">
-          <tr>
-            <td width="100%" class="header" style="padding-top: 10px;" colspan="2">
-              <table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#FFFFFF">
-                <tr>
-                </tr>
-              </table>
-            </td>
-          </tr>
-          <tr>
-            <td width="75%" style="font-family: Helvetica, Arial, sans-serif;">
-              <p style="font-weight: 400; font-size: 16px; line-height: 1; margin: 10px 0 10px 0;">&nbsp;</p>
-            </td>
-            <td width="25%">&nbsp;</td>
-          </tr>
-          <tr>
-            <td bgcolor="#00823B" style="font-family: Helvetica, Arial, sans-serif; margin: 0 0 30px 0;">
-              <p style="font-size: 36px; font-weight: bold; text-align: center; color: #ffffff; line-height: 1.315789474; margin: 30px 0px 15px 0px;">
-                <%= t(".heading.heading") %>
-              </p>
+  <body style="font-family: Helvetica, Arial, sans-serif; font-size: 16px; margin: 0; color: #0b0c0c">
```

Comment:
More a question than feedback. Isn't the `<body>` tag at the same level as the `<head>` tag, and if so shouldn't

- the indentation be applied to both or
- the indentation should match the `<head>` tag